src/Scheduler.java

```diff
@@ -0,0 +1,34 @@
+import java.util.ArrayList;
+
+public class Scheduler implements Runnable {
+	private ArrayList<FloorEvent> floorEvents;
+	private ArrayList<ArrivalEvent> arrivalEvents;
+	private int floor;
+	private MiddleMan box;
+	private MiddleMan box2;
+	
+	/**
+	 * Public constructor to create Scheduler object and instantiate instance variables
+	 */
+	public Scheduler(MiddleMan box, MiddleMan box2) {
+		this.floorEvents = new ArrayList<FloorEvent>();
+		this.arrivalEvents = new ArrayList<ArrivalEvent>();
+		this.box = box;
+		this.box2 = box2;
+		floor = -1;
+	}
```

Comment:
@krishane2018 you never use or set the floor variable btw

Reply:
I don't think the floor variable is necessary for this iteration.